projects/netgrif-components/src/lib/navigation/navigation-double-drawer/navigation-double-drawer.component.html

```diff
@@ -31,19 +31,22 @@
 
             <mat-menu #userRailMenu="matMenu">
                 <div mat-menu-item role="menuitem" class="large-menu-item" fxLayout="row" fxFlexAlign="center center"
-                    [routerLink]="profileRouterLink" routerLinkActive="active">
+                     [routerLink]="profileRouterLink" routerLinkActive="active">
                     <mat-icon class="margin-auto" fxFlex>face</mat-icon>
                     <div class="large-menu-item" fxLayout="column" fxFlexAlign="start center" fxFlex>
-                        <span *ngIf="!user.isImpersonating()"  class="large-menu-item-text" fxFlex="50">{{user.fullName}}</span>
-                        <span *ngIf="user.isImpersonating()" class="large-menu-item-text" fxFlex="50">{{user.fullName}} ({{user.getSelfOrImpersonated().fullName}})</span>
+                        <span *ngIf="!user.isImpersonating()" class="large-menu-item-text"
+                              fxFlex="50">{{user.fullName}}</span>
+                        <span *ngIf="user.isImpersonating()" class="large-menu-item-text" fxFlex="50">{{user.fullName}}
```

Comment:
the two `<span>`s chould be simplified with an ngIf-else construct

Reply:
It could be. But this construct is used in more places so it will be refactored in the future altogether.